### DISCUSSION TITLE
Spreading list items across multiple columns

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3521] [Q&A] [answered]
Consider the following `typst` document, where I am trying to spread the items of a list. Unfortunately, all items show in the first column (and do not spread). Could you please help me? Thanks!

```
#let items = (
  [`0`: Monday;],
  [`0`: Monday;],
  [`0`: Monday;],
  [`0`: Monday;],
)

#block(
  height: 2em,
  grid(columns: (1fr, 1fr),
  gutter: 0.8em,
  list(..items))
)

```

**Comment by andreasKroepelin** [comment]
Consider using
```typ
grid(columns: (1fr, 1fr),
  gutter: 0.8em,
  ..items.flatten())
```
instead.

**Reply by psads-git** [reply]
Thanks, @andreasKroepelin, for your reply, which almost works. In fact, now the items do spread, but the bullets of the list are missing! Is there some workaround?

**Reply by andreasKroepelin** [reply, marked as answer]
Ah yes sorry. Try:

```typ
grid(columns: (1fr, 1fr),
  gutter: 0.8em,
  ..items.flatten().map(list.item))
```

**Reply by psads-git** [reply]
Thanks, @andreasKroepelin: that is it!
